File: idades.c
#include <stdio.h>
#include <string.h>
char nome1,nome2 [50];
int idade1,idade2;
double media;
int main()
{
    printf("Digite o nome da primeira pessoa: ");
    scanf("%s", &nome1);
    printf("Digite a idade dessa pessoa: ");
    scanf("%d", &idade1);
    printf("Digite o nome da segunda pessoa: ");
    scanf("%s", &nome2);
    printf("Digite a idade dessa pessoa: ");
    scanf("%d", &idade2);
    media=(idade1+idade2)/2;
    printf("A idade media entre essas pessoas e: %.1lf\n", media);
    return 0;
}

File: L1, ling. C#/idades.cs
using System;
using System.Globalization;
class Program {
    static void Main() {
        int idade1, idade2;
        string nome1, nome2;
        double media;
        CultureInfo CI = CultureInfo.InvariantCulture;
        Console.Write("Nome da primeira pessoa: ");
        nome1 = Console.ReadLine();
        Console.Write("Idade: ");
        idade1 = int.Parse(Console.ReadLine());
        Console.Write("Nome da segunda pessoa: ");
        nome2 = Console.ReadLine();
        Console.Write("Idade: ");
        idade2 = int.Parse(Console.ReadLine());
        media = (idade1 + idade2) / 2.0;
        Console.WriteLine("A idade média de {nome1} e {nome2} é de {media:F1} anos");
    }
}
